bin/tasks.js

```diff
@@ -4,12 +4,12 @@ const path = require('path');
 const cwd = process.cwd();
 const target = path.resolve(cwd, 'public');
 const source = path.resolve(__dirname, '../src');
-const sass = require('witch')('node-sass');
+const npmsass = require('npm-sass');
```

Comment:
> personally, I would prefer all of this to be in just plain ol boring JS code

I'm inclined to agree, not least because there's absolutely zero error handling code in this setup at the moment. If things fail you get a nice console.error, but the process doesn't actually fail (i.e. exits with 0)